### PR TITLE
Propagate TLSSecurityProfile to CDI

### PR DESCRIPTION
**Pull request by tiraboschi**
Propagate TLSSecurityProfile to CDI.
Add unit tests.

Signed-off-by: Simone Tiraboschi <stirabos@redhat.com>

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly
  
**Release note**:

```release-note
Propagate TLSSecurityProfile to CDI
```

### CODE REVIEW

File: controllers/operands/cdi_test.go [hunk 1]
```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"time"
 
+	openshiftconfigv1 "github.com/openshift/api/config/v1"
```
Comment (machadovilaca) [inline controllers/operands/cdi_test.go:7]: nit: group this with other openshift imports

Comment (tiraboschi) [reply controllers/operands/cdi_test.go:7]: done

**Comment by tiraboschi** [comment]
ignoring roundings on coverage
/override coverage/coveralls

**Pull request merged**